.github/stale.yml

```diff
@@ -28,20 +28,18 @@ staleLabel: stale
 
 # Comment to post when marking as stale. Set to `false` to disable
 markComment: >
-  This issue has been automatically marked as stale because it has no recent
-  activity in the last 60 days. It will be closed in 14 days if no further
-  activity occurs. If this issue is still relevant please leave a comment to
-  let us know and the stale label will be automatically removed. Thank you for
-  your contributions.
+  Thank you for your contribution! Since there hasn't been any recent activity,
+  this issue has been automatically marked as stale, and may eventually be closed.
+  Please bump this issue by leaving a comment if you want to keep it open.
 
 # Comment to post when removing the stale label.
 # unmarkComment: >
 #   Your comment here.
 
 # Comment to post when closing a stale Issue or Pull Request.
 closeComment: >
-  This issue has been stale for 14 days and is now closed due to inactivity, if
-  this issue is still relevant please re-open this issue, or open a new issue.
+  This stale issue is now closed due to inactivity. Please re-open or re-file if
+  if the issue is still relevant. Thank you!
```

Comment:
ditto I prefer to state the reasoning here, and the reason is that its been stale for over 30 days.